spynnaker/pyNN/models/neuron/additional_inputs/additional_input_ca2_adaptive.py

```diff
@@ -37,9 +37,15 @@ class AdditionalInputCa2Adaptive(AbstractAdditionalInput):
 
     def __init__(self,  tau_ca2, i_ca2, i_alpha):
         r"""
-        :param float tau_ca2: :math:`\tau_{\mathrm{Ca}^{+2}}`
-        :param float i_ca2: :math:`I_{\mathrm{Ca}^{+2}}`
-        :param float i_alpha: :math:`I_{\alpha}`
+        :param tau_ca2: :math:`\tau_{\mathrm{Ca}^{+2}}`
+        :type tau_ca2: Float, iterable of Floats, RandomDistribution or \
```

Comment:
I think our standard way of writing something like this would be something like "float, iterable(float), ~pyNN.random.RandomDistribution or (mapping) function" given how other documentation currently looks.  (I think we'd put pyNN.random.RandomDistribution because our RandomDistribution class is a simple wrapper to the PyNN class).

Reply:
fixed